src/metabase/driver.clj

```diff
@@ -695,7 +695,10 @@
     ;; Whether the driver supports loading dynamic test datasets on each test run. Eg. datasets with names like
     ;; `checkins:4-per-minute` are created dynamically in each test run. This should be truthy for every driver we test
     ;; against except for Athena and Databricks which currently require test data to be loaded separately.
-    :test/dynamic-dataset-loading})
+    :test/dynamic-dataset-loading
+
+    ;; Whether the driver is officially supported by Metabase's query parser.
+    :sql-parsing})
```

Comment:
The reverse dependency here actually feels quite bad to me. Whether this is true depends more on the version of Metabase than on the version of the driver.

Reply:
Decided not to expose any public API for opting in yet. There is just a private multimethod.